apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -226,9 +216,34 @@ type OutputDefaults struct {
 	Elasticsearch *Elasticsearch `json:"elasticsearch,omitempty"`
 }
 
-// ClusterLogForwarderList is a list of ClusterLogForwarders
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+
+// ClusterLogForwarder is an API to configure forwarding logs.
+//
+// You configure forwarding by specifying a list of `pipelines`,
+// which forward from a set of named inputs to a set of named outputs.
+//
+// There are built-in input names for common log categories, and you can
+// define custom inputs to do additional filtering.
+//
+// There is a built-in output name for the default openshift log store, but
+// you can define your own outputs with a URL and other connection information
+// to forward logs to other stores or processors, inside or outside the cluster.
+//
+// For more details see the documentation on the API fields.
 //
-// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type ClusterLogForwarder struct {
```

Comment:
Based on https://github.com/openshift/cluster-logging-operator/pull/1048/files#r675382725 we need to ensure we do not lose any of these annotations

Reply:
annotations reverted